test-coverage/src/test/java/org/eclipse/collections/impl/CoverageTest.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2016 Goldman Sachs.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Eclipse Distribution License v. 1.0 which accompany this distribution.
+ * The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v10.html
+ * and the Eclipse Distribution License is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ */
+
+package org.eclipse.collections.impl;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CoverageTest
+{
+    @Test
+    public void smoke()
+    {
+        Assert.assertTrue("Smoke Test required by JaCoCo to aggregate reports", true);
```

Comment:
Yep. Addressed in https://github.com/Godin/eclipse-collections/commit/40c9b98417e9a7ea8b0be8b9a884d6a1ebf16151